### ISSUE TITLE
Issues with Discord, and Steam

### ISSUE THREAD

**Issue by chanouh** [meetfranz/franz#544]
### Current Behavior

My mic on discord dont work correclty, when I use the real discord app and a real navigator for web version of discord, it's work correclty. So please fix it, for now I can't use Discord on Franz.

I dont have another microphone to test sorry.

For Steam, when I add someone on my friend list on the real Steam, I need to close franz and launch franz again to get my friendlist Steam updated on franz.

Possible solutions : 
Maybe use the real discord app and just use franz like a window for discord.

I'm running a laptop with Windows 10 64 bits.

Thank you,

chanou.h@gmail.com

**Comment by closingin** [comment]
I can confirm this bug, as I'm running Arch Linux and have the same problem, so it's not related to @chanouh 's Windows installation. Looking at my terminal output, an error occurs when I join a vocal channel.

```
[11036:0129/173526.814927:ERROR:audio_send_stream.cc(93)] Failed to set up send codec state.
```

This error doesn't seem to be Franz-related though, this issue has appeared on [electron's issues](https://github.com/electron/electron/issues/8991) and wire ones [here](https://github.com/wireapp/wire-desktop/issues/612) and [here](https://github.com/wireapp/wire-desktop/issues/1106). Apparently it's a bug with WebRTC, so I think that you can remove the `service` label.

**Comment by cauepor** [comment]
Same problem here, I user MacOS High Sierra v. 10.13.3 and Franz 5.0.0-beta.17, when I join a discord channel I can't hear the people in it or speak, when I check the discord voice tab it shows no input from  my mic.

Bonus note: Whatsapp integration seems to be suffering some problems when playing voice messages and video messages, it just gets stucked when play is hit.

**Comment by Trixarian** [comment]
Been having the same issue lately. My mic would be muted when I start up Franz and you couldn't click the icon to unmute it. In my case, going into User Settings---> Voice and Video somehow magically unmutes the mic. Afterwards you can mute and unmute the mic as usual. Works fine in the official app and web interface in chrome (although for some reason both also start muted too), I'm running it on a Windows 7 Ultimate 32bit laptop

**Comment by featherbear** [comment]
![image](https://user-images.githubusercontent.com/1159091/41981268-e014504a-7a6b-11e8-8cb8-2fbe07ec1b66.png)

I'm getting a slightly different issue... the audio devices aren't appearing at all!
Also the checkboxes for some other options are missing

**Comment by Zeioth** [comment]
I can confirm it. My issue is the same.

**Comment by DrCox1911** [comment]
I have the same issue with the latest Discord update, the console output when trying to speak is:
`[253:875] [8260] (audio_send_stream.cc:519): Unable to create encoder for {name: opus, clockrate_hz: 48000, num_channels: 1, parameters: {stereo: 0}}
[253:875] [8260] (audio_send_stream.cc:261): Failed to set up send codec state.`

**Comment by XLPmc** [comment]
Same problem here! Log: `[000:000] [8697] (audio_send_stream.cc:519): Unable to create encoder for {name: opus, clockrate_hz: 48000, num_channels: 1, parameters: {stereo: 0, usedtx: 0, useinbandfec: 0}}
[000:001] [8697] (audio_send_stream.cc:261): Failed to set up send codec state.
[000:077] [8697] (audio_send_stream.cc:519): Unable to create encoder for {name: opus, clockrate_hz: 48000, num_channels: 1, parameters: {stereo: 0, usedtx: 0, useinbandfec: 1}}
[000:077] [8697] (audio_send_stream.cc:261): Failed to set up send codec state.
[000:077] [8697] (audio_device_generic.cc:31): BuiltInAECIsAvailable: Not supported on this platform
[000:107] [8697] (audio_send_stream.cc:519): Unable to create encoder for {name: opus, clockrate_hz: 48000, num_channels: 1, parameters: {stereo: 0, usedtx: 0, useinbandfec: 1}}
[000:107] [8697] (audio_send_stream.cc:261): Failed to set up send codec state.
`

**Comment by brianmego** [comment]
Confirmed from Gentoo:

```
[000:000] [4673] (audio_send_stream.cc:519): Unable to create encoder for {name: opus, clockrate_hz: 48000, num_channels: 1, parameters: {stereo: 0}}
[000:001] [4673] (audio_send_stream.cc:261): Failed to set up send codec state.
```

This was from opening Discord 0.0.8, User Settings -> Voice & Video -> Let's Check

Further, I do have libopus.so.0.6.1 installed (md5sum 4c1da7cf83c7b7cf8b3c366afa7b6530)

However, I do not see it in dynamically linked libraries for the Discord binary:

```
ldd Discord | grep opus
```

**Comment by olealgoritme** [comment]
sooo, how do we enable the microphone in discord standalone app?

**Comment by brianmego** [comment]
For new it ended up being a red herring, with the recent discord release
being the issue:

https://support.discordapp.com/hc/en-us/articles/360022307532--Known-Issues-Audio-Input-issues-on-Linux-client
<https://www.google.com/url?q=https://support.discordapp.com/hc/en-us/articles/360022307532--Known-Issues-Audio-Input-issues-on-Linux-client&sa=D&source=hangouts&ust=1548818128776000&usg=AFQjCNFWG6ICbPjlLLtMRfTCzfYjRj27fg>

**Comment by RikGeboers** [comment]
same problem still, is there a solution?

**Comment by tlsimmons88** [comment]
I am running Windows 10.  Franz version 5.1.0

When using Franz and Discord.  People can hear me talk.  But I am unable to hear them.
I have ensured the audio settings are set to use my headset for both input and output.

If I use discord outside of Franz with the same headset it works just fine.

**Comment by hcp-coder** [comment]
There is a community edition called Ferdi. It's a fork of this project but is actively maintained and works amazingly. Give it a go: https://github.com/getferdi/

Unfortunatetly Franz does not seem to be developed actively. There is no support contact for paying customers. It seems that someone is maintaining it since there are offering discounts from time to time.

The mic and video problem is solved with the community edition (v5.4.3).

I would like to support Franz but not sure if it is maintained anymore,
